### PR TITLE
fix: integrate changes requested by CCSL-2020

### PR DESCRIPTION
**Pull request by danse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,22 +10,30 @@ JSON files listing locations in
...
+in the JSON files. Whenever data-models is referenced in the
+documentaiton of the project, it referes to the [informal-conventional
+data-model we are using for the JSON files in](./json/README.md)
 
-Adding a country requires 3 steps:
+### Adding countries
 
-1. Add a `country_name.json` file to the `json` folder
-(read the `README.md` in there, too).
+Adding a country requires to add a `country_name.json` file to the
```
Comment (twitchett) [inline README.md]: Almost! "Adding a country requires adding a `country_name.json` file"

Comment (danse) [inline README.md]: oh, _to add_ is wrong? I didn't know :thinking:

**Comment by danse** [comment]
@adamvert ?

**Comment by danse** [comment]
also @adamvert i assume that you are not interested in reviewing the pull requests that will have to follow this in order to update the dependencies for `angular-locations` and the call centre

**Comment by adamvert** [comment]
Just add me as a reviewer anyway, although I probably won't look too closely

**Pull request merged**